addons/mail/static/src/utils/common/misc.js

```diff
@@ -91,3 +91,27 @@ export function compareDatetime(date1, date2) {
     }
     return date1.ts - date2.ts;
 }
+
+/**
+ * Compares two version strings.
+ *
+ * @param {string} v1 - The first version string to compare.
+ * @param {string} v2 - The second version string to compare.
+ * @return {number} -1 if v1 is less than v2, 1 if v1 is greater than v2, and 0 if they are equal.
+ */
+export function compareVersions(v1, v2) {
+    const parts1 = v1.split(".");
+    const parts2 = v2.split(".");
+
+    for (let i = 0; i < Math.max(parts1.length, parts2.length); i++) {
+        const num1 = parseInt(parts1[i]) || 0;
+        const num2 = parseInt(parts2[i]) || 0;
+        if (num1 < num2) {
+            return -1;
+        }
+        if (num1 > num2) {
+            return 1;
+        }
+    }
+    return 0;
+}
```

Comment:
This is "fine" as a util function. But I think it'd easier to use `<`/`=`/`>=`/`greaterThan()`/`equals()` / etc. rather than think in terms of negative / 0 / positive.

I get `.sort()` works that way, but this is far from natural and whenever we use it we still have to refresh the difference between negative and positive.

Reply:
Renamed to `isGreaterVersion` which is enough for my needs :)